docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -1,6 +1,13 @@
 [[elastic-agent-container]]
 = Run {agent} in a container
 
+****
+**Reviewers:** Is there any reason why we don't link to the topic about
+Elastic Agent environment variables like we do in the K8s topic? In fact, in the
+K8s topic, we include some of the setting descriptions under "configuration details"
+What should we do in this topic? Just include a link?
```

Comment:
I think including a link to that same environment variables topic would be appropriate here. You're right that it's a similar scenario as the K8's topic.